### PR TITLE
Add Qwik / web framework #5844

### PR DESCRIPTION
**Pull request by nurbek91**
### website:
https://github.com/builderio/qwik
### examples:
https://www.builder.io/

### CODE REVIEW

File: src/technologies/q.json [hunk 1]
```diff
@@ -237,5 +237,27 @@
...
+      },
+      "div": {
+        "attributes": {
+          "q:obj": "\\;confidence:50",
+          "q:host": "\\;confidence:50"
+        }
+      }
+    },
+    "oss": true,
+    "website": "https://github.com/builderio/qwik"
```
Comment (mhevery) [inline src/technologies/q.json:261]:
```suggestion
    "website": "https://qwik.builder.io"
```

I know it is not public just yet, but should be soon.

File: src/technologies/q.json [hunk 2]
```diff
@@ -237,5 +237,27 @@
...
+  "Qwik": {
+    "cats": [
+      18
+    ],
+    "description": "Qwik is designed for the fastest possible page load time, by delivering pure HTML with near 0 JavaScript.",
+    "icon": "Qwik.svg",
+    "dom": {
+      "html": {
+        "attributes": {
+          "q:version": "^([\\d\\.]+(?:-\\d+)?)\\;version:\\1"
```
Comment (mhevery) [inline src/technologies/q.json:250]: The `q:version` can be on any element not just on `html` Is there a way to express it?

File: src/technologies/q.json [hunk 3]
```diff
@@ -237,5 +237,27 @@
...
+        }
+      },
+      "div": {
+        "attributes": {
+          "q:obj": "\\;confidence:50",
```
Comment (mhevery) [inline src/technologies/q.json:255]: It is possible that there is no `q:obj` if the content is static.
There should always be at least on `q:host` attribute so I think that is fine. But why not just rely on `q:version`?

Comment (nurbek91) [reply src/technologies/q.json:251]: Hey @mhevery  , added **body** tag.
Versions should now be checked for all these tags **html,body,div**
But then my knowledge ends there.
Further @AliasIO , i need your help mate.

Comment (AliasIO) [reply src/technologies/q.json]: I've updated it to always just use `q:version`, assuming it's always present (on any element).

**Pull request merged**